src/deck-options.md

```diff
@@ -422,91 +422,80 @@ number, and recommend you keep it lower than 0.95 and higher than the [minimum r
+
+Use this option sparingly, as it will add a review entry to each of your cards, and increase the size of your collection.
 
 If you wish to visualize how FSRS would change your schedule without altering your workload, there are two ways you can do so:
 
+- Create a backup, enable FSRS with rescheduling, check the future due graph in statistics, and then undo or restore from the backup if necessary.
+
 - Enable FSRS without rescheduling, and compare the interval and stability graphs. The
 interval graph will show the current intervals of cards; the stability graph will show the
 intervals FSRS would give cards if the desired retention is 0.9.
 
-- Create a backup, enable FSRS with rescheduling, check the future due graph, and then undo or restore from the backup if needed.
-
 ### Compute Minimum Recommended Retention
 
-This experimental tool assumes you're starting with 0 cards, and will
-attempt to calculate the amount of material you'll be able to retain
-in the given time frame. The estimated retention will greatly depend
-on your inputs, and if it significantly differs from 0.9, it's a sign
-that the time you've allocated each day is either too low or too high
-for the amount of cards you're trying to learn. This number can be
-useful as a reference, but it is not recommended to copy it into the
-desired retention field.
+attempt to find the desired retention value that leads to the most material learnt, in the least amount of time. To accurately simulate your learning process, this feature requires a minimum of 400+ reviews. The calculated number can serve as a reference when deciding what to set your desired retention to. You may wish to choose a higher desired retention, if you’re willing to trade more study time for a greater recall rate. However, setting your desired retention lower than the minimum is not recommended, as you'll spend more time studying than necessary.
 
 #### Learning and Re-learning Steps
 
 (Re)learning steps of 1+ days are not recommended when using FSRS. The main 
-reason they were popular with the old SM-2 scheduler is because repeatedly 
+reason they were popular with the legacy SM-2 scheduler is because repeatedly 
 failing a card after it has graduated from the learning phase could reduce 
 its ease a lot, leading to what some people called "ease hell". This is not 
 a problem that FSRS suffers from.  By keeping your learning steps under a 
 day, you will allow FSRS to schedule cards at times it has calculated are 
-optimum for your material and memory.  Another reason not to use longer 
+optimum for your material and memory. Another reason not to use longer 
 learning steps is because FSRS may end up scheduling the first review for a 
-shorter time than your last learning step, leading to the `Hard` button 
-showing a longer time than `Good`.
+shorter time than your last learning step, leading to the "Hard" button 
+showing a longer time than "Good".
```

Comment:
The Manual is really inconsistent. I've searched the word "Good" in my PDF and in some cases it's bold, otherwise in code format, sometimes with quotes. I'll open an issue about this.